Dockerfile

```diff
@@ -22,18 +23,18 @@ RUN npm run build
 FROM elixir-build AS release
 COPY --from=assets-build /build /build
 WORKDIR /build
-ENV MIX_ENV=prod
```

Comment:
Is this removal safe? Does the `mix release` still use the expected value?

Reply:
I got this as a suggestion while pairing with @stefanotorresi but after checking it a bit more I think you are right. I'll re-add for now